src/aead/aes_gcm_truncated_tag_96.txt

```diff
@@ -0,0 +1,230 @@
+[encrypt]
```

Comment:
At the top of this test input file and at the top of the other test input file, include a note that says that these test vectors come from the Chromium test suite, and include the path to the test within the Chromium repo.